### ISSUE TITLE
mamba fails to install python 3.9

### ISSUE THREAD

**Issue by shaunc** [mamba-org/mamba#971]
I am trying to upgrade python to 3.9. Conda hangs; following a [tip here](https://github.com/conda/conda/issues/9367#issuecomment-793968239) -- I decided to try mamba:

> mamba install python=3.9

However, this gives:
```
  - package anaconda-custom-py38_1 requires python >=3.8,<3.9.0a0, but none of the providers can be installed
```
I presume this is because mamba is not trying to update conda itself. Is this a bug? Is there a workaround if not?

**Comment by wolfv** [comment]
Is because you're using some specific anaconda package that probably doesn't have a python 3.9 version. Please remove the indicated package and try again

**Comment by shaunc** [comment]
I tried that. I get:
```
Removing specs: ['anaconda-custom-py38_1']
Transaction

  Prefix: /home/shauncutts/anaconda3

  Nothing to do


PackagesNotFoundError: The following packages are missing from the target environment:
  - anaconda-custom-py38_1
```
From https://stackoverflow.com/questions/37782008/conda-packages-with-version-name-of-custom I see that this is a dependency of conda itself. Is it safe to remove? If so -- shouldn't mamba automatically remove it during the upgrade?

**Comment by wolfv** [comment]
sorry, the proper package to remove is actually `anaconda`.

**Comment by wolfv** [comment]
anyways, it's almost always better to start with miniconda or miniforge / mambaforge: https://github.com/conda-forge/miniforge and then create new environments seperate from the base env.

**Comment by shaunc** [comment]
Thanks!

Ffor the moment I was able to create a sub environment for py3.9. However, I did want to be able to update my base environment. If I need py3.8 for a particular project I can have a sub-environment for that, but wanted to roll forward what the base was, in order to cut down on the number of things I was maintaining. 

For particular projects, I use poetry anyway, but often I want some "ambient" things available as binaries which are inconvenient to install and/or to optimize using poetry. Then as time passes, I want to update all living projects to use a new ambient environment. (Or at least consciously decide to leave them behind/specialize them for some reason.)

If I do tear out conda and install mini forge, can I maintain a base environment like this?
